pytest/tests/sanity/resharding.py

```diff
@@ -15,7 +15,7 @@
 from configured_logger import logger
 from cluster import get_binary_protocol_version, init_cluster, load_config, spin_up_node
 from utils import MetricsTracker, poll_blocks
-from resharding_lib import append_shard_layout_config_changes, get_epoch_offset, get_genesis_num_shards, get_genesis_shard_layout_version, get_target_num_shards, get_target_shard_layout_version
+import resharding_lib
```

Comment:
nit: how about `from resharding_lib import *`?

Reply:
Personally don't think it's a great idea to do `import *`, it's considered as bad practice due to potential collision of namespace and difficulty in locating the import module.

In case you don't like the `resharding_lib.abc`, I can change this back to `from resharding_lib import abc, pqr` etc.

Reply:
Yeah I prefer having the actual code as short as possible and I don't care about long imports.